### DISCUSSION TITLE
Level tag for spans

### DISCUSSION THREAD

**Discussion by mTsBucy1** [tokio-rs/tracing-opentelemetry#177] [Q&A] [answered]
Hi there,

I have noticed that in my setup, where I configure `tracing-opentelemetry` as a tracing subscriber and sending the data via OTLP to a jaeger all-in-one instance, I cannot see the level of the spans anymore. If the span happens to contain events, then they contain the tag `"level"` which is then set to the event level. No such field exists for the span.
Having this field would be quite nice if I could then filter out some TRACE spans and only see the INFO spans, for a quicker glance of what happened.

So I was wondering whether this is intended functionality, a bug, an oversight leading to a feature request, or some artefact of using jaeger to display these span?

Thanks

**Comment by djc** [comment]
By "anymore", do you mean there is a regression, or is this particular to sending OTLP to Jaeger?

I think usually people set a level filter for their subscriber as a way to strip out the TRACE Spans?

**Reply by mTsBucy1** [reply]
I mean "anymore" in the sense of dataflow: at the `tracing` level, this information exists (and is filtered by me), but I still want to collect more data than I want to look at normally.

Unless there is a technical or normative reason to strip this information, it seems restrictive to remove it when sending OLTP messages.

**Reply by mladedav** [reply, marked as answer]
OpenTelemetry doesn't define levels for either spans or events. However, it's true that we could add the level as a custom span attribute, similarly to how it's added to events.
